fcrepo-search-api/src/main/java/org/fcrepo/search/api/SearchParameters.java

```diff
@@ -74,6 +78,14 @@ public int getMaxResults() {
         return conditions;
     }
 
+    /**
+     * Returns the list of fields to display in the results.
+     * @return
+     */
+    public List<Condition.Field> getFields() {
+        return fields;
+    }
+
     @Override
     public String toString() {
```

Comment:
Should probably update the `toString` to include the additional fields.

Reply:
Will do.